instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/test/java/io/lumigo/javaagent/instrumentation/awssdk/v1_11/SnsTracingTest.java

```diff
@@ -251,99 +251,6 @@ void snsNotificationTriggersSqsMessage() {
                                             Assertions.assertThat(value).contains(PUBLISH_RESPONSE);
                                             Assertions.assertThat(value).contains("<MessageId>");
                                           });
-                                }),
-                    span ->
-                        span.hasName("SQS.ReceiveMessage")
```

Comment:
Why exactly this got deleted? before we tested sns & sqs now this test only check sns.

Reply:
Because of changing the business logic inside open-telemetry repository. 
Attached the screenshot with commit where this span is not generated anymore. 

<img width="1394" alt="Снимок экрана 2024-08-27 в 11 50 06 AM" src="https://github.com/user-attachments/assets/82d3999c-ea03-4089-af93-2410b38715f3">

Reply:
On the right side you can see that this span code is deleted.